sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaExactlyOnceSink.java

```diff
@@ -520,6 +521,7 @@ void commitTxn(long lastRecordId, Counter numTransactions) throws IOException {
       }
     }
 
+    @SuppressFBWarnings("REC_CATCH_EXCEPTION")
```

Comment:
The goal of the JIRA is to fix the warnings, in this case deal with the different exceptions. Can't this be fixed?
If it is really impossible to fix, prefer using the `findbugs-filter.xml` (or `spotbugs-filter.xml`) file to do the exclusions.

Reply:
Yes, you are right, fixed by changing type of caught exception